versioned_docs/version-3.0.0-beta/framework-guide/deploy-vanilla.md

```diff
@@ -0,0 +1,64 @@
+---
+id: deploy-vanilla
+title: How to Deploy Vanilla Apps in less than 30 Seconds
+sidebar_label: Deploy Vanilla App
+slug: /framework-guide/deploy-vanilla
+---
+
+This guide will show you how to deploy a Vanilla project and set up your domain.
+
+[Vanilla JS](http://vanilla-js.com/) is a fast, lightweight, cross-platform framework for building incredible, powerful JavaScript applications.
+
+## Follow these steps to deploy your  Vanilla app on Spheron successfully.
+
+### Step 1: Put Your Vanilla Project Code in a Git Repo
+
+While this feature is not new to developers, the magic of Spheron is that every push to any branch in your GitHub, GitLab, or Bitbucket repo triggers a new build and deployment of your project.
+
+### Step 2: Update Your Vanilla Project for Deploying on Decentralised Storage
```

Comment:
This step is not required for vanilla app

Reply:
i removed this already,just realise i didnt push

Reply:
Ok then, np!